src/app/shared/services/layout.service.ts

```diff
@@ -8,6 +8,12 @@ export interface FullFrameContent {
   animated: boolean,
 }
 
+interface ConfigureOptions {
+  expanded: boolean,
+  canToggle?: boolean,
+  showTopRightControls?: boolean,
```

Comment:
Shouldn't `canToggle` be `canToggleLeftMenu` or `canToggleFullFrameContent` or whatever ?

Reply:
fixed